### ISSUE TITLE
some confusions

### ISSUE THREAD

**Issue by leileilin** [dhlee347/pytorchic-bert#32]
if I pretrain Bert with masking strategy，In principle, I can predict the word of mask given by a source sentence and a target sentence with mask. 
anyone can tell me how to do that?
thank you man.
